src/components/Card.js

```diff
@@ -1,27 +1,73 @@
 import styled from 'styled-components';
 
-export default function PlantCard({ name, fact, water, spot, info }) {
+export default function PlantCard({ name, fact, water, spot, info, img }) {
   return (
     <Card>
-      <Name>{name}</Name>
-      <ul>
+      <Name>
+        <PlantIcon>
+          <svg
+            width="40"
+            height="40"
+            fill="green"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M23.2692 1.06583C14.6367 7.2275 17.1467 18.1042 17.1467 18.1042L19.0192 14.5583L23.2692 1.06583ZM23.2692 1.06583C26.8067 11.0642 18.5133 18.5342 18.5133 18.5342L19.0133 14.5558L23.2692 1.06583ZM8.18333 3.00417C12.645 3.29583 16.3633 7.83833 16.3633 7.83833C15.2496 10.4603 14.9443 13.3551 15.4867 16.1517C11.3533 12.5325 7.52 8.77917 8.1825 3.00417H8.18333ZM0.456667 16.9317C5.54893 20.2687 11.3712 22.3278 17.4292 22.9342C14.3975 15.9167 9.01 13.4008 0.456667 16.9317V16.9317ZM18.6242 22.8958C27.8117 22.43 35.5342 17.0525 35.5433 17C27.7267 13.6292 22.0492 15.5008 18.6242 22.8958Z"
+              stroke="#4ECB71"
+              stroke-width="0.833333"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            />
+          </svg>
+        </PlantIcon>
+        {name}
+      </Name>
+      <PlantPic>{img}</PlantPic>
+      <List role="list">
```

Comment:
unside a `ul` dont use `p` elements (its not allowed - https://caninclude.glitch.me/caninclude?child=p&parent=ul) 

you might wanna use description list (`dl`) or include the `p`in the `li `element.